docker-compose.yml

```diff
@@ -1,19 +1,41 @@
 version: '3.1'
 
 services:
-  # Dependencies for running unit tests
+  # Dependencies for running unit tests and for running the 'irma' command line tool using one of the test configurations.
   postgres:
     image: postgres:13-buster
     environment:
       POSTGRES_USER: testuser
       POSTGRES_PASSWORD: testpassword
       POSTGRES_DB: test
+    networks:
+      - irma-net
     ports:
       - 5432:5432
+  postgres-init:
+    image: postgres:13-buster
+    environment:
+      PGHOST: postgres
+      PGUSER: testuser
+      PGPASSWORD: testpassword
+      PGDATABASE: test
+    networks:
+      - irma-net
+    depends_on:
+      - postgres
+    volumes:
+      - ./server/keyshare/schema.sql:/schema.sql
+      - ./server/keyshare/cleanup.sql:/cleanup.sql
+    # We have to wait until the database is up and running.
+    # Database might already be running, so we might need to do a cleanup first.
+    command: /bin/sh -c "sleep 5 && (psql -f schema.sql || (psql -f cleanup.sql && psql -f schema.sql))"
```

Comment:
Couldn't we instead do [`DROP SCHEMA IF EXISTS irma CASCADE;`](https://www.postgresql.org/docs/8.2/sql-dropschema.html), and then simplify this to the following?

```sh
/bin/sh -c "sleep 5 && psql -f cleanup.sql && psql -f schema.sql"
```

Reply:
Yes we can, I'd propose to change this in the `cleanup.sql` script then. Otherwise we duplicate this logic.